Switched to black on white and white on black for the demos (#123)
MicrosoftEdge/Demos: pwa-manifest-localization/README.md

# Manifest Localization for Web Apps

➡️ **[Open the demo](https://microsoftedge.github.io/Demos/pwa-manifest-localization/)** ⬅️

> This demo showcases the Web App Manifest `x_localized` members feature, which allows Progressive Web Apps to provide localized app metadata (such as name, description, and icons) for different languages directly in the manifest file.

Reviewer: ```suggestion
This demo showcases the use of `_localized` suffixed web app manifest members, which allow Progressive Web Apps (PWAs) to provide localized app metadata (such as name, description, and icons) for different languages directly in the manifest file.
```

> The demo supports English, German, and Arabic localizations.

Reviewer: ```suggestion
The demo app is localized in English, German, and Arabic.
```

> ## How to use

Reviewer: ```suggestion
## How to use the demo app
```

> 1. Open the [demo site](https://microsoftedge.github.io/Demos/pwa-manifest-localization/) in a supported browser.

Reviewer: ```suggestion
1. In Microsoft Edge, open the [Manifest Localization Test App](https://microsoftedge.github.io/Demos/pwa-manifest-localization/) in a new window or tab.
```

> 2. Install the PWA in one of the supported languages (English, German, or Arabic). The PWA will display localized values.

Reviewer: ```suggestion
2. In the address bar, click the **App available** button to install the PWA on your device.

    If your browser is set to English, German, or Arabic, the installed PWA displays the name, short name, icon, description, and shortcuts that are defined in the web app manifest file for that language.
```

> 3. To see the localization's app update flow in action, try switching your browser's Preferred Language settings to another supported language. On relaunch, you will be prompted to update the installed PWA to the new localized values.

Reviewer: ```suggestion
3. In Microsoft Edge, go to `edge://settings/languages`.

4. Under **Preferred languages**, click **...** next to English, German, or Arabic, choosing a language that your browser isn't already using.

5. Select **Display Microsoft Edge in this language**.

6. Restart the PWA.

   You are prompted to update the installed PWA to the new localized values.
```

# Manifest Localization for Web Apps

➡️ **[Open the demo](https://microsoftedge.github.io/Demos/pwa-manifest-localization/)** ⬅️

This demo showcases the use of `_localized` suffixed web app manifest members, which allow Progressive Web Apps (PWAs) to provide localized app metadata (such as name, description, and icons) for different languages directly in the manifest file.

The demo app is localized in English, German, and Arabic.

> ## How to use the demo app
> 
> 1. In Microsoft Edge, open the [Manifest Localization Test App](https://microsoftedge.github.io/Demos/pwa-manifest-localization/) in a new window or tab.
> 2. In the address bar, click the **App available** button to install the PWA on your device.
> 
>     If your browser is set to English, German, or Arabic, the installed PWA displays the name, short name, icon, description, and shortcuts that are defined in the web app manifest file for that language.
> 3. In Microsoft Edge, go to `edge://settings/languages`.
> 
> 4. Under **Preferred languages**, click **...** next to English, German, or Arabic, choosing a language that your browser isn't already using.
> 
> 5. Select **Display Microsoft Edge in this language**.
> 
> 6. Restart the PWA.
> 
>    You are prompted to update the installed PWA to the new localized values.

Reviewer: ```suggestion
1. In Microsoft Edge, open the [Manifest Localization Test App](https://microsoftedge.github.io/Demos/pwa-manifest-localization/) in a new window or tab.

2. In the address bar, click the **App available** button to install the PWA on your device.

    If your browser is set to English, German, or Arabic, the installed PWA displays the name, short name, icon, description, and shortcuts that are defined in the web app manifest file for that language. 

3. In Microsoft Edge, go to `edge://settings/languages`.

4. Under **Preferred languages**, click **...** next to either English, German, or Arabic, choosing the one which your browser isn't already using.

5. Select **Display Microsoft Edge in this language**.

6. Restart the PWA.

   You are prompted to update the installed PWA to the new localized values.
```

# Manifest Localization for Web Apps

➡️ **[Open the demo](https://microsoftedge.github.io/Demos/pwa-manifest-localization/)** ⬅️

This demo showcases the use of `_localized` suffixed web app manifest members, which allow Progressive Web Apps (PWAs) to provide localized app metadata (such as name, description, and icons) for different languages directly in the manifest file.

The demo app is localized in English, German, and Arabic.

## How to use the demo app

1. In Microsoft Edge, open the [Manifest Localization Test App](https://microsoftedge.github.io/Demos/pwa-manifest-localization/) in a new window or tab.

2. In the address bar, click the **App available** button to install the PWA on your device.

>     If your browser is set to English, German, or Arabic, the installed PWA displays the name, short name, icon, description, and shortcuts that are defined in the web app manifest file for that language. 

Reviewer: ```suggestion
   If your browser is set to English, German, or Arabic, the installed PWA displays the name, short name, icon, description, and shortcuts that are defined in the web app manifest file for that language. 
```
